### PR TITLE
GraphQL: Update index page

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) updates the GraphQL front page to align with the current 2.4 development priorities.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/graphql/index.html

### CODE REVIEW

File: src/guides/v2.3/graphql/index.md [hunk 1]
```diff
@@ -18,13 +18,14 @@ As of Magento 2.3.4, GraphQL provides the following features:
...
-*  All new GraphQL coverage will rely on newly-defined Storefront APIs. If there are no appropriate Storefront APIs for the particular business scenario being covered, these APIs will be introduced. Existing coverage will work with the Storefront APIs without changing the current schema.
-*  Storefront API implementations must rely on the Magento Framework only.  No dependencies on entities defined in Magento modules will be allowed.
-*  GraphQL resolvers will not contain business logic.
-
-The Catalog, Customer, and Cart modules will be the first modules to be re-architected. Details about the new architecture and modules will be published as it becomes available, before the 2.4 release.
+For the 2.4 release, Magento teams are working toward completing GraphQL coverage for Business to Consumer (B2C) uses cases, with emphasis on the following:
+
+*  Order history for logged in customers
+*  Reorders
+*  Replace product-specific mutations that add products to a card with a single mutation that can handle all product types
```
Comment (jeff-matthews) [inline src/guides/v2.3/graphql/index.md]: card or cart?

Comment (keharper) [reply src/guides/v2.3/graphql/index.md]: D'oh. Well, it passes spell check.

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
@dshevtsov  I don't know why those two files showed up

**Comment by dshevtsov** [comment]
> @dshevtsov I don't know why those two files showed up

they were part of [this commit](https://github.com/magento/devdocs/pull/6834/commits/462be4d9545c8311fb921eb8556b785d8812ce0b)

**Comment by dshevtsov** [comment]
> Yikes, I cam't believe I missed that in my initial review.

@jeff-matthews nevertheless, looks like you approved it again 😄

**Comment by keharper** [comment]
Ah, I created those files when Dima was showing me the latest rake tool. Since GH didn't show any changed text, I thought it was a weird hiccup.

**Comment by jeff-matthews** [comment]
what?! i could've sworn i clicked comment :|

**Comment by keharper** [comment]
@dshevtsov @jeff-matthews  The `test.md` files are gone.

**Comment by dshevtsov** [comment]
@keharper please re-request a review after you add fixes

**Comment by keharper** [comment]
running tests

**Pull request merged**